README.md

```diff
@@ -1,3 +1,9 @@
 # sm-plugin-ServerCommandFilter
 - Filters point_servercommand->Command() using user-defined rules to restrict maps.
 - Filters all convars changes performed via VScript
+- Filters all commands performed via VScript
+
+> [!IMPORTANT]
+> Upgrade from 1.x to 2.x introduce a config change in sourcemod/configs.
+> Rename: PointServerCommandFilter.cfg -> PointServerCommandFilter.cfg
```

Comment:
i think you meant ServerCommandFilter.cfg

Reply:
yes, nice catch, thanks!